### DISCUSSION TITLE
Help understanding recipe section for Admin Screens

### DISCUSSION THREAD

**Discussion by TaylorEdm** [OrchardCMS/OrchardCore#7536] [Q&A] [answered]
I am currently trying to understand how to modify the recipe file similar to what you have for the example for the Blogs Recipe. A top-level admin menu that is specific to the content type (See screenshot of blog recipe )
![image](https://user-images.githubusercontent.com/10617595/98390010-75a63f00-201a-11eb-9b9c-a4bc7e3fab9d.png) 

From the blog recipe, I found four sections I believe I need to add to my recipe to work:

**Variables:**  
`"variables": {
    "topLevelAdminMenuId": "[js:uuid()]",
 }`

**Content Items:**
`"ContentParts": [
        {
          "Name": "topLevelAdminMenu",
          "Settings": {},
          "ContentPartFieldDefinitionRecords": [
            {
              "FieldName": "MediaField",
              "Name": "Image",
              "Settings": {
                "ContentPartFieldSettings": {
                  "DisplayName": "Banner Image",
                  "Position": "0"
                },
                "ContentIndexSettings": {
                  "Included": false,
                  "Stored": false,
                  "Analyzed": false,
                  "Sanitized": false,
                  "Tokenized": false,
                  "Template": null
                },
                "MediaFieldSettings": {
                  "Multiple": false
                }
              }
            }
          ]
        }
}`

** Content Types:**
`"ContentTypes": [       
        {
          "Name": "topLevelAdminMenu",
          "DisplayName": "Custom top level menu",
          "Hidden": false,
          "Settings": {
            "ContentTypeSettings": {
              "Creatable": true,
              "Draftable": true,
              "Versionable": true,
              "Listable": true,
              "Securable": true
            }
          },
          "ContentTypePartDefinitionRecords": [
            {
              "PartName": "TitlePart",
              "Name": "TitlePart",
              "Settings": {
                "ContentTypePartSettings": {
                  "Position": "0"
                }
              }
            },
            {
              "PartName": "AutoroutePart",
              "Name": "AutoroutePart",
              "Settings": {
                "ContentTypePartSettings": {
                  "Position": "1"
                },
                "ContentIndexSettings": {
                  "Included": false,
                  "Stored": false,
                  "Analyzed": false,
                  "Sanitized": false,
                  "Tokenized": false,
                  "Template": null
                },
                "AutoroutePartSettings": {
                  "AllowCustomPath": true,
                  "Pattern": "{{ Model.ContentItem | display_text | slugify }}",
                  "ShowHomepageOption": true
                }
              }
            }
}`

**Admin Menu:**
`{
      "name": "AdminMenu",
      "data": [
        {
          "Id": "baef6f85ad13481681cde70ada401333", 
          "Name": "Admin menu ",
          "Enabled": true,
          "MenuItems": [
            {
              "$type": "OrchardCore.AdminMenu.AdminNodes.LinkAdminNode, OrchardCore.AdminMenu",
              "LinkText": "Landing Page",
              "LinkUrl": "[js: 'Admin/Contents/ContentItems/' +  variables('topLevelAdminMenuId') + '/Display']",
              "IconClass": "fas fa-rss",
              "UniqueId": "7b293d57056a4eebb3713f07f12c65d8",
              "Enabled": true,
              "Text": null,
              "Id": null,
              "Href": null,
              "Url": null,
              "Position": null,
              "LinkToFirstChild": true,
              "LocalNav": false,
              "Culture": null,
              "Resource": null,
              "Items": [],
              "RouteValues": null,
              "Permissions": [],
              "Classes": []
            }]
		}]
	}`

**Content:**
`{
  "name": "Content",
  "data": [
	{
	  "ContentType": "topLevelAdminMenu",
	  "ContentItemId": "[js: variables('topLevelAdminMenuId')]",
	  "DisplayText": "Blog",
	  "Latest": true,
	  "Published": true,
	  "Owner": "[js: parameters('AdminUsername')]",
	  "Author": "[js: parameters('AdminUsername')]",
	  "TitlePart": {
		"Title": "Thing"
	  },
	  "HtmlBodyPart": {
		"Html": "<p>stuff</p>"
	  },
	  "AutoroutePart": {
		"Path": "landingPage"
	  },
	  "ListPart": {},
	  "Blog": {
		"Image": {
		  "Paths": [
			"Logo.png"
		  ]
		}
	  }
	}]
}`

I believe I am missing something with the setup of the Recipe files on the setup. It seems like there has been a content item associated (which requires a content type). Or maybe I am misunderstanding the example. The documentation on this section seems to be a bit too [high level](https://docs.orchardcore.net/en/dev/docs/reference/modules/AdminMenu/). 

TLDR: I am lost on modifying the Admin Recipe section to create a custom menu to display a specific content type. I am using version 1.0.0-rc2-13450

**Comment by TaylorEdm** [comment, marked as answer]
You gotta enable Admin Menus by putting "OrchardCore.AdminMenu" in the enabled section of the recipe script.
